wp-content/plugins/hurumap/src/IndicatorBlock/Edit.js

```diff
@@ -0,0 +1,159 @@
+            value={sourceTitle}
+            onChange={val => {
+              setAttributes({ sourceTitle: val });
+            }}
+          />
+          <TextControl
+            label="Source Link"
+            value={sourceLink}
+            onChange={val => {
+              setAttributes({ sourceLink: val });
+            }}
+          />
+          <SelectControl
+            label="Layout"
+            value={layout}
+            options={[
+              { label: 'Select Layout', value: undefined },
+              { label: 'Document', value: 'document' },
+              { label: 'Image', value: 'image' },
+              { label: 'HTML', value: 'html' }
+            ]}
+            onChange={val => {
+              setAttributes({ layout: val });
+            }}
+          />
+          {layout && (
+            <Fragment>
+              {layout === 'html' ? (
+                <TextareaControl
+                  label="Add html"
+                  value={src}
+                  onChange={val => {
+                    setAttributes({ src: val });
+                  }}
+                  placeholder={__('Paste HTML')}
+                  aria-label={__('HTML')}
+                />
+              ) : (
+                <MediaPlaceholder
+                  onSelect={el => {
+                    setAttributes({ src: el.url, srcId: el.id });
+                  }}
+                  accepts={layout === 'image' ? 'image/*' : '.pdf'}
+                  value={{ id: srcId }}
+                  labels={
+                    layout === 'image'
+                      ? {
+                          title: __('Image'),
+                          instructions: __(
+                            'Upload media or pick one from library'
+                          )
+                        }
+                      : {
+                          title: __('Document'),
+                          instructions: __(
+                            'Upload file or pick one from library'
+                          )
+                        }
+                  }
+                />
+              )}
+            </Fragment>
+          )}
+        </PanelBody>
+      </InspectorControls>
+
+      <InsightContainer
+        hideInsight
+        title={title}
+        description={description}
+        variant="analysis"
+        actions={{ handleDownload: null }}
+        source={
+          sourceLink || sourceTitle
+            ? {
+                title: sourceTitle,
+                href: sourceLink
+              }
+            : undefined
+        }
+      >
+        <div />
+        <>
+          {layout && layout === 'html' && src && <SandBox html={src} />}
+          {layout && layout === 'image' && src && (
+            <img src={src} alt="indicator" />
+          )}
+          {layout && layout === 'document' && src && (
+            <PDFDataContainer title={title} source={src} />
+          )}
+        </>
+      </InsightContainer>
+    </Fragment>
```

Comment:
1. Lets arrange them as how they're rendered:
   1. Title,
    2. Actual content.
    3. Source, and
    4. Description

2. Not sure `Layout` is a good name for what we're using it for. `Indicator Type` or something similar would be better IMHO.

3. Not sure but don't we need to set both `accepts` & `allowedTypes`? [This](https://github.com/WordPress/gutenberg/tree/master/packages/block-editor/src/components/media-placeholder) _kinda_ suggests that since they each affect the behaviour of a different component/part.